pattern-1/README.md

```diff
@@ -0,0 +1,94 @@
+# DC/OS Resources for WSO2 API Manager Deployment Pattern 1
+
+## Quick Start Guide
```

Comment:
I think it would be better to use the latest mysql connector v5.1.45 than using an older version. WDYT?

Reply:
+1 A great thought @ctienshi! I will try it out and update! Thanks!

Reply:
The following issue was created to track this: https://github.com/wso2/dcos-apim/issues/4